crates/sargon/src/gateway_api/models/logic/response/ledger_state.rs

```diff
@@ -1,6 +1,7 @@
 use crate::prelude::*;
 
-#[cfg(test)]
+// TODO: Uncomment the following line once we don't need to mock response of GatewayClient::get_pre_authorization_status()
+//#[cfg(test)]
```

Comment:
is this accurate? are we still mocking?

Reply:
should have heard my own comment: uncommenting line 4